binding.gyp

```diff
@@ -114,21 +114,37 @@
             ['OS == "win"', {
               'msvs_settings': {
                 'VCCLCompilerTool': {
+                  'ExceptionHandling': 2,       # /EHsc
                   # 0 - MultiThreaded (/MT)
                   # 1 - MultiThreadedDebug (/MTd)
                   # 2 - MultiThreadedDLL (/MD)
                   # 3 - MultiThreadedDebugDLL (/MDd)
                   'RuntimeLibrary': 3,
                   'AdditionalOptions': [
                     '-std:c++17',
+                    "/DEBUG",
+
+                    # Uncomment to enable address sanitizer
```

Comment:
Why is disabled by default? Maybe add a comment explaining this.

Reply:
Because address sanitizers have some performance overhead and also are not available on all compilers.